src/util.js

```diff
@@ -322,25 +322,15 @@ define([
             }
             return value;
         }
-        opts = opts || {};
-        if (opts.normalize_xmlns) {
-            var xmlns = $(xml.parseXML(serverForm)).find('data').xmlAttr('xmlns');
-            localForm = localForm.replace(/(data[^>]+xmlns=")(.+?)"/,
-                                    '$1' + xmlns + '"');
-        }
```

Comment:
Just confirming that it was the case that nowhere called `xmlDiff` with opts provided and therefore this code is irrelevant, right?

Reply:
Yep, the only call to `xmlDiff` is the one updated in this PR.